fahrschein/src/main/java/org/zalando/fahrschein/NakadiClient.java

```diff
@@ -109,6 +109,26 @@ public SubscriptionBuilder subscription(String applicationName, Set<String> even
         return new SubscriptionBuilder(this, applicationName, eventNames);
     }
 
+    /**
+     * Delete subscription based on subscription ID.
+     *
+     * @param subscriptionId
+     * @return
+     * @throws IOException
+     */
+    public boolean deleteSubscription(String subscriptionId) throws IOException {
```

Comment:
In all failure cases I would expect an exception, which means this can be a `void` method